<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>322</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>322</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16000</width>
    <height>16000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Rule</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>11</y>
     <width>470</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>470</width>
     <height>300</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IP_Address">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Port">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Action">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Action">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>Allow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Block</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Cancel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OK">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP_Address">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UseYourIp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Use your IP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_IP_Address</tabstop>
  <tabstop>lineEdit_Port</tabstop>
  <tabstop>comboBox_Action</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
